src/components/container.jsx

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+var React = require('react');
+var componentMixin = require('./mixins/componentMixin');
+var multiMixin = require('./mixins/multiMixin');
+var FormioComponent = require('../FormioComponent');
+
+module.exports = React.createClass({
+    displayName: 'Container',
+    mixins: [componentMixin, multiMixin],
```

Comment:
Containers can't be multiple so we should remove the multiMixin and conver getSingleElement to getElements().